cmd/magneticow/api.go

```diff
@@ -150,7 +159,7 @@ func (h *ApiReadmeHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// Because .nfo files are right padded with \x00'es.
 	content = bytes.TrimRight(content, "\x00")
 
-	w.Header().Set("Content-Type", "text/plain;charset=UTF-8")
+	w.Header().Set(ContentType, ContentTypeHtml)
```

Comment:
I think this should remain `text/plain` and not become `text/html`

Reply:
my bad :100: